### DISCUSSION TITLE
how check index_file_size for a created collection

### DISCUSSION THREAD

**Discussion by rocke-dong** [milvus-io/milvus#16507] [Q&A and General discussion]
Dear All,
milvus 2.0.2
collection = Collection(name=collection_name, schema=schema, using='default', shards_num=2, index_file_size=2048)
could I  check whether it take effects for assigned index_file_size?

quations2, Is the default index_file_size is in milvus.yaml ?
# Related configuration of rootCoord, used to handle data definition language (DDL) and data control language (DCL) requests
rootCoord:
 ......
  minSegmentSizeToEnableIndex: 1024 # It's a threshold. When the segment size is less than this value, the segment will not be indexed

thansk!!

**Comment by yhmo** [comment]
In v2.0, the segment size is defined in milvus.yml
```
  segment:
    maxSize: 512 # Maximum size of a segment in MB
```

In sdk, the "index_file_size" parameter doesn't work.

The "minSegmentSizeToEnableIndex: 1024" means milvus will not build index for segment whose rows less than 1024.
